payjoin/src/receive/mod.rs

```diff
@@ -3,268 +3,26 @@
 //! This module contains types and methods used to receive payjoin via BIP78.
 //! Usage is pretty simple:
 //!
-//! 1. Generate a pj_uri [BIP 21](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki) using [`payjoin::Uri`](crate::Uri)::from_str
+//! 1. Generate a pj_uri [BIP 21](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki)
```

Comment:
this is one of a few line break changes, to keep the text aligned
previously it looked like this
<img width="1411" alt="Screenshot 2024-01-31 at 22 54 44" src="https://github.com/payjoin/rust-payjoin/assets/21229375/1e1d2d74-970e-448b-be0d-496bbef54927">

now it like this
<img width="1383" alt="Screenshot 2024-01-31 at 22 53 03" src="https://github.com/payjoin/rust-payjoin/assets/21229375/cc2d85e7-aca8-479c-815a-7e136978e13a">
9de">